### PR TITLE
Writing xlsx was simple with apache poi

### PR DESCRIPTION
**Pull request by gaurav-dogra**
Do you think I can add unit tests somewhere in WriteOutputXlsx.java

### CODE REVIEW

File: src/main/java/gmailgdogra/ExcelToExcelParsingWebApp.java [hunk 1]
```diff
@@ -13,7 +13,10 @@ public static void main(String[] args) throws IOException {
         SpringApplication.run(ExcelToExcelParsingWebApp.class, args);
         List<SwipeRecord> outputData =
                 PrepareOutputData.from("src/main/resources/testFile.xlsx");
-        outputData.forEach(System.out::println);
+        boolean isSuccess = WriteOutputXlsx.write(outputData);
+        if (isSuccess) {
```
Comment (alfabravo2013) [inline src/main/java/gmailgdogra/ExcelToExcelParsingWebApp.java:17]: While using boolean is ok, explore the modern approach: you use a special class for a result of operation. For example, if the operation succeeds, Result<T> will contain the result of the operation and if it fails, Result<T> contains the reason of the failure (usually an exception). This makes life easier.

File: src/main/java/gmailgdogra/WriteOutputXlsx.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                "Logical Device"), 0); // title row
+        writeData(outputData);
+        autoSizeColumns();
+        String date = LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-yyyy"));
+
+        try (FileOutputStream outputStream = new FileOutputStream("Securitas Report " + date + ".xlsx")) {
+            workbook.write(outputStream);
+        }
+
+        return true;
```
Comment (alfabravo2013) [inline src/main/java/gmailgdogra/WriteOutputXlsx.java:30]: Generally, there are a few different approaches how to check a result of an operation: return a boolean; return void on success or throw an exception on failure; return a special class with the result which content will be checked at the call site. You use a mixed approach: return true or throw an exception. It's a bit inconsistent.

**Review by alfabravo2013 — APPROVED** [review]

**Comment by alfabravo2013** [comment]
> 
> 
> Do you think I can add unit tests somewhere in WriteOutputXlsx.java

You have the only public method, so you can (and should) test it. If you have doubts how to test it, look at that method's return. It returns true on success and throws an exception on failure. These are your test cases.

File: src/main/java/gmailgdogra/ExcelToExcelParsingWebApp.java (hunk 1 above)
Comment (gaurav-dogra) [reply src/main/java/gmailgdogra/ExcelToExcelParsingWebApp.java:17]: I'll look into it

File: src/main/java/gmailgdogra/WriteOutputXlsx.java (hunk 2 above)
Comment (gaurav-dogra) [reply src/main/java/gmailgdogra/WriteOutputXlsx.java:30]: I'll look into it

**Comment by gaurav-dogra** [comment]
The suggested improvements  are no longer needed as that particular pieces of codes are no longer in the codebase

**Pull request closed without merging**